### PR TITLE
Create playbook for Cluster Observability Operator UIPlugins deployment

### PR DESCRIPTION
**Pull request by SurajGudaji**

### CODE REVIEW

File: playbooks/roles/ocp-coo/tasks/e2e.yaml [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+                project: "openshift-config-managed"
+              spec:
+                plugin:
+                  kind: "PrometheusDatasource"
+                    spec:
+                      direct_url: "https://thanos-querier.openshift-monitoring.svc.cluster.local:9091"
+
+    - name: Download dashboard config file
+      get_url:
+        url: "https://raw.githubusercontent.com/lihongyan1/coo_test/main/prometheus.json"
```
[the remark this answers is not in the document]
Comment (SurajGudaji) [reply playbooks/roles/ocp-coo/tasks/e2e.yaml:127]: when we tested with the prometheus-dashboard-example.yaml file from the main repo the dashboard plugin in UI was not seen, that's way we are using lihongyan1's Prometheus configuration file.

File: examples/all.yaml [hunk 2]
```diff
@@ -375,11 +375,24 @@ bgp_address:
...
+enable_distributed_tracing_uiplugin: false
+enable_troubleshootingpanel_uiplugin: false
+coo_role_enable: false
 coo_namespace: "openshift-observability-operator"
 coo_catalogsource_name: "coo-catalog"
 coo_catalogsource_image: "brew.registry.redhat.io/rh-osbs/iib:760399"
 coo_channel: "development"
+coo_e2e_namespace: "coo-e2e"
+coo_logstack_namespace: "openshift-logging"
+minio_image: "quay.io/rh-ee-jdarapu/minio/minio"
```
[the remark this answers is not in the document]
Comment (SurajGudaji) [reply examples/all.yaml:391]: yes we can, at the time of development of this playbook the minio image was only available in docker hub, so we had pushed the image to jeevan's account and using it.

File: playbooks/roles/ocp-coo/README.md [hunk 3]
```diff
@@ -6,9 +6,15 @@ This role is used to deploy Cluster Observability Operator.
 Requirements
 ------------
 
-- OCP 4.x healthy cluster on PowerVS.
+- OCP 4.12+ healthy cluster on PowerVC.
```
[the remark this answers is not in the document]
Comment (SurajGudaji) [reply playbooks/roles/ocp-coo/README.md]: No can also be done on PowerVS

File: playbooks/roles/ocp-coo/README.md [hunk 4]
```diff
@@ -18,18 +24,32 @@ Role Variables
...
 | coo_catalogsource_image                | no       | "brew.registry.redhat.io/rh-osbs/iib:760399" | Custom catalog source index image for Cluster Observality Operator |
 | coo_channel                    | no       | development                                                                               | Operator upgrade channel                                                                                                       |
+| distributed_tracing_qe_repo                    | no       | https://github.com/openshift/distributed-tracing-qe.git                                                                               | Git Repo to deploy a TempoStack instance                                                                                                       |
+| distributed_tracing_qe_repo_branch                    | no       | main                                                                               | Git branch to deploy a TempoStack instance                                                                                                       |
```
[the remark this answers is not in the document]
Comment (SurajGudaji) [reply playbooks/roles/ocp-coo/README.md:38]: They are made default in defaults/main.yaml file

File: examples/all.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (SurajGudaji) [reply examples/all.yaml]: Done

File: playbooks/roles/ocp-coo/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (SurajGudaji) [reply playbooks/roles/ocp-coo/README.md]: Done

File: playbooks/roles/ocp-coo/tasks/e2e.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (varad-ahirwadkar) [reply playbooks/roles/ocp-coo/tasks/e2e.yaml:127]: ok

File: playbooks/roles/ocp-coo/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (varad-ahirwadkar) [reply playbooks/roles/ocp-coo/README.md:39]: OK

File: playbooks/roles/ocp-coo/vars/main.yaml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
 
 coo_e2e_env:
-  PATH: "{{ ansible_env.PATH }}:/usr/local/go/bin"
+  PATH: "{{ ansible_env.PATH }}:/usr/local/chainsaw"
```
Comment (yussufsh) [inline playbooks/roles/ocp-coo/vars/main.yaml:4]: Are we not using the variable `chainsaw_path`?

Comment (SurajGudaji) [reply playbooks/roles/ocp-coo/vars/main.yaml]: ahh yes, I have made the changes, can you please review the changes.
Thanks.

**Review by yussufsh — DISMISSED** [review]
/approve

**Review by varad-ahirwadkar — APPROVED** [review]

**Review by yussufsh — APPROVED** [review]
/lgtm

**Pull request merged**